src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -435,6 +451,18 @@ export default class MultipleCategoryBarPlot extends React.Component<
         }
     }
 
+    private setInitialSelectedOption = () => {
```

Comment:
this is a bit of an anti-pattern.  calling from the child component to update the state of parent component on render.  it should be possible to do this in parent component prior to mount?

Reply:
In the parent component, it doesn't contain data in the form of `IMultipleCategoryBarPlotData`, from which I have to extract all the unique major categories for dropdown options. Therefore, to avoid extra calculations, I passed the prop from the parent and updated it when it became available in the child.